doc/community-build/docker/cortx-all/README.md

```diff
@@ -57,9 +62,10 @@ This document provides step-by-step instructions to build required binaries and
        ```
 
 3. Run the following command to build the CORTX packages.
+  
   - For rocky linux use below command:
    ```
-   docker run --rm -v /var/artifacts:/var/artifacts -v /mnt/cortx:/cortx-workspace ghcr.io/seagate/cortx-build:rockylinux-8.4 make clean cortx-all-rockylinux-image cortx-ha
+   docker run --rm -v /var/artifacts:/var/artifacts:Z -v /mnt/cortx:/cortx-workspace:Z ghcr.io/seagate/cortx-build:rockylinux-8.4 make clean cortx-all-rockylinux-image cortx-ha
```

Comment:
AFIK `Z` option needed to make the bind mount content is private and unshared.  Refer - https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label 
We are expecting SELinux to be disabled and we are only spawning a single temporary container. So do we need to specify the `Z` option explicitly.

Reply:
yes reverted as earlier it was returning "permission denied" error